src/components/menu/examples/menu-open-sub-menu-programmatically.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+    MenuItem,
+    ListSeparator,
+    LimelMenuCustomEvent,
+} from '@limetech/lime-elements';
+import { Component, State, h, Host } from '@stencil/core';
+import { CascadingMenuItems } from './item-constants';
+
+/**
+ * Opening sub-menus programmatically
+ *
+ * **This example is currently not in use because it's an experimental feature**
+ *
+ * It is possible to open any sub-menu in the menu-hierarchy.
+ * This is done by using the parentItem property of the MenuItem class.
+ * @link item-constants.ts
+ */
+@Component({
+    tag: 'limel-example-menu-open-sub-menu-programmatically',
+    shadow: true,
+    styleUrl: 'menu-open-sub-menu-programmatically.scss',
+})
+export class MenuOpenSubMenuProgrammaticallyExample {
+    private readonly rootItems: Array<MenuItem | ListSeparator> =
+        CascadingMenuItems;
+
+    @State()
+    private items: Array<MenuItem | ListSeparator> = this.rootItems;
+
+    @State()
+    private lastSelectedItem: string;
+
+    @State()
+    private currentSubMenu: MenuItem;
+
+    @State()
+    private openMenu: boolean = false;
+
+    public render() {
+        return (
+            <Host>
+                {this.renderMenu()}
+                <limel-example-value
+                    label="Last selected item"
+                    value={this.lastSelectedItem}
+                />
+            </Host>
+        );
+    }
+
+    private renderMenu() {
+        return (
+            <div class="menu-container">
+                <limel-menu
+                    items={this.items}
+                    open={this.openMenu}
+                    currentSubMenu={this.currentSubMenu}
+                    onSelect={this.handleSelect}
+                    onNavigateMenu={async (
+                        event: LimelMenuCustomEvent<MenuItem>
+                    ) => {
+                        if (!event.detail) {
+                            this.items = this.rootItems;
+                        }
+                    }}
+                    onCancel={() => {
+                        this.items = this.rootItems;
+                        this.openMenu = false;
+                    }}
```

Comment:
We don't need to change anything here, since we're currently not using this example anyway, but we don't want people to use arrow-functions or `.bind()` in templates, because both of them cause a new function to be created every time the template is re-rendered, which eats up resources and performance. And since we don't want people to do that, we also really don't want to teach them to do it by doing it in the code examples 😊

Reply:
We add the file, so I think it's a good idea to fix it anyways. It's quick.